bundles/org.openhab.binding.tr064/src/main/java/org/openhab/binding/tr064/internal/FritzboxActions.java

```diff
@@ -76,16 +95,67 @@ public class PhonebookActions implements ThingActions {
         } else {
             int matchCountInt = matchCount == null ? 0 : matchCount;
             if (phonebook != null && !phonebook.isEmpty()) {
-                return handler.getPhonebookByName(phonebook).flatMap(p -> p.lookupNumber(phonenumber, matchCountInt))
-                        .orElse(phonenumber);
+                return Objects.requireNonNull(handler.getPhonebookByName(phonebook)
+                        .flatMap(p -> p.lookupNumber(phonenumber, matchCountInt)).orElse(phonenumber));
             } else {
                 Collection<Phonebook> phonebooks = handler.getPhonebooks();
-                return phonebooks.stream().map(p -> p.lookupNumber(phonenumber, matchCountInt))
-                        .filter(Optional::isPresent).map(Optional::get).findAny().orElse(phonenumber);
+                return Objects.requireNonNull(phonebooks.stream().map(p -> p.lookupNumber(phonenumber, matchCountInt))
+                        .filter(Optional::isPresent).map(Optional::get).findAny().orElse(phonenumber));
             }
         }
     }
 
+    @RuleAction(label = "create configuration backup", description = "Creates a configuration backup")
```

Comment:
@J-N-K, @lolodomo - what is your opinion about I18N action labels/descriptions? Since they are not supported, in my opinion we should have no such requirement, but we could accept I18N since it seems not to be shown anywhere in the UI, and to be future-ready. WDYT?

See https://github.com/openhab/openhab-addons/pull/14134#issuecomment-1374644503

Just looking for establishing some guidelines for the time being in this area.

Reply:
Yes, until https://github.com/openhab/openhab-core/issues/1745 is resolved and actions are enabled in UI these annotations are not used.

Reply:
Thanks for the reference and by that confirming. Question was about I18N texts for label/description though. In #14134 @lolodomo commented the lack of them, so I added them, but the REST API showed that they are not supported yet either. So I think it doesn't matter much if we are adding them now. Since some bindings have them, which means we can't add actions to the UI without also adding I18N support.

Reply:
Well my understanding is: If the annotations are non-i18n, then their value is used, otherwise the localized version is used. So in any case having a plain text here should never cause issues (except that i18n is not working). I would consider it a bug if it is REQUIRED to have i18 here.

That aside: it does not matter at all until UI support for rules is implemented. Once this is the case, we can re-visit the add-ons that support actions.

Reply:
Agreed. 😄